src/mmt-odk/src/info/kwarc/mmt/odk/SCSCP/Example/ClientExample.scala

```diff
@@ -9,7 +9,7 @@ object ClientExample {
     import info.kwarc.mmt.odk.OpenMath._
 
     // establish a connection
-    val client = SCSCPClient("scscp.gap-system.org")
+    val client = SCSCPClient("localhost")
```

Comment:
It's just an example - it's technically not even supposed to be in there. Seems more like a test class than something that should be in the final jar. I' too stupid to figure out how to merge it into devel, though